master/Makefile

```diff
@@ -74,6 +77,7 @@ pre-package:
 	cp ../common/dist/*.whl build/wheels/
 	cp ../cli/dist/*.whl build/wheels/
 	cp ../harness/dist/*.whl build/wheels/
+	cp $(shell which growforest) build/
```

Comment:
Is this portable? Like if I run this on a mac wouldn't this package the GOOS=darwin version of growforest into a Linux container for the master?

Reply:
I'm kinda new to Go dependency management / packaging so I'm very open to suggestions. I wasn't too concerned about it because we don't build packages from people's workstations, do we? I assumed that was done in CI on an OS we choose very carefully for this. (and for context - I'm speaking like I wrote this line because I did - this commit is a mix of stuff I did to get Katie started, and I've actually been working on a fork of this PR doing some of the same things you suggest here :D)